### ISSUE TITLE
Service not running on 1.7.9

### ISSUE THREAD

**Issue by 3x1at1on** [Dielee/volvo2mqtt#80]
**Describe the bug**
Service is not running on 1.7.9. service starts and the terminates.

**Version info**:<br>
HA addon

**Logs**:<br>
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
Traceback (most recent call last):
  File "/volvoAAOS2mqtt/main.py", line 2, in <module>
    from volvo import authorize
  File "/volvoAAOS2mqtt/volvo.py", line 3, in <module>
    import mqtt
  File "/volvoAAOS2mqtt/mqtt.py", line 6, in <module>
    import util
  File "/volvoAAOS2mqtt/util.py", line 9, in <module>
    from const import units
  File "/volvoAAOS2mqtt/const.py", line 115, in <module>
    {"name": "Distance to Service", "domain": "sensor", "id": "km_to_service", "unit": "km" if not units.get(settings["babelLocale"]) else units[settings["babelLocale"]]["distance_to_service"]["unit"], "icon": "wrench-clock", "url": VEHICLE_DIAGNOSTICS_URL},
                                                                                                                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'distance_to_service'
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped`

**Comment by Dielee** [comment]
My bad, I will fix this tomorrow

**Comment by Dielee** [comment]
Fixed with version v1.7.10
